packages/bpk-tokens/gulpfile.babel.js

```diff
@@ -123,6 +124,8 @@ const createTokens = done => {
         }),
       )
       .on('error', done)
+      .pipe(transformDarkValues())
```

Comment:
It seems like the wrong level of abstraction to do this after the files have been serialized to JSON rather than as part of the process in Theo. The responsbility is now spread in different places instead of being in Theo.

Reply:
Yeah - we know this isn't ideal but theo doesn't support this. I tried to write a transformer but theo only allows you to handle a single value in a transformer 😢

Reply:
I would rather have this logic in the app repos, which has similar logic given that limitation in Theo.

Reply:
Yeah - at the time it seemed nicer to keep it in one place, but I dunno...

Reply:
I don't like the idea of having 3 repos doing the same task when it could just be one..